packages/ActionBar/stories/ShowcaseApp/MySort.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Sort, useSort } from "../../src";
+
+export default function MySort({ columns }) {
```

Comment:
👏 this is way more consistent than our previous iteration awesome :) hooks to the rescue.